plugins/api/src/atomic_apply_metadata_operations.cpp

```diff
@@ -188,12 +210,24 @@ namespace
         }
 
         try {
-            const auto& db_plugin_config = config.at(irods::CFG_PLUGIN_CONFIGURATION_KW).at(irods::PLUGIN_TYPE_DATABASE);
+            auto db_plugin_config = std::move(config.at(irods::CFG_PLUGIN_CONFIGURATION_KW).at(irods::PLUGIN_TYPE_DATABASE));
```

Comment:
instead of capturing the json object, this should just capture the property name as a string